packages/tables/src/views/OverflowButton.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableStyles from '@zendeskgarden/css-tables';
+import { retrieveTheme } from '@zendeskgarden/react-theming';
+import { composeEventHandlers } from '@zendeskgarden/react-selection';
+
+const COMPONENT_ID = 'tables.overflow_button';
+
+/**
+ * Accepts all `<button>` props
```

Comment:
May render correctly as a `<div role="button">` – although CSS should probably provide resets.

Reply:
This is now fixed. I opened an issues against [css-tables](https://github.com/zendeskgarden/css-components/issues/79) for us to address there.